torchdata/datapipes/iter/util/rows2columnar.py

```diff
@@ -50,7 +50,9 @@ class Rows2ColumnarIterDataPipe(IterDataPipe[Dict]):
     """
     column_names: List[str]
 
-    def __init__(self, source_datapipe: IterDataPipe[List[Union[Dict, List]]], column_names: List[str] = None) -> None:
+    def __init__(
+        self, source_datapipe: IterDataPipe[List[Union[Dict, List]]], column_names: Optional[List[str]] = None
+    ) -> None:
```

Comment:
Another n00b q: what formatter are you using in torchdata vs torch.utils.data? Is linting part of CI on the github side?

Reply:
Formatter being used - black - https://github.com/pytorch/data/blob/4725a01ce38c646b6273acacb9df895b3d9c3f66/.pre-commit-config.yaml#L28-L34.

linting part of CI is on the github side. I think there is an explicit exclusion of it in phabricator.